### DISCUSSION TITLE
m2r2

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#1303] [Q&A] [answered]
Is m2r2 used? I saw an entry that said a recent commit had been deleted.
Is the description of m2r2 below omitted from deletion?

https://github.com/jertel/elastalert2/blob/master/docs/README.md

```
pip install m2r2 sphinx_rtd_theme sphinx
```

https://github.com/jertel/elastalert2/blob/master/requirements-dev.txt#L4

```
m2r2
```

**Comment by jertel** [comment, marked as answer]
Yes, I've removed m2r2 as it is conflicting with the new ReadTheDocs version that went into effect late September. I discovered yesterday that the ElastAlert 2 docs have not been building for over 3 weeks, as a result of that RTD upgrade. After reviewing the RTD release notes and making the configuration changes required I then found that the m2r2 was causing older versions of Sphinx to be installed. Since only the FAQ used the m2r2 module I modified the FAQ.rst and FAQ.md to remove the markdown formatting and converted it to pure RST.

I've removed the two references you posted above. Thanks for the reminder.
